src/main/java/io/vlingo/xoom/http/PostRequestBody.java

```diff
@@ -0,0 +1,31 @@
+package io.vlingo.xoom.http;
+
+
+import io.vlingo.xoom.http.media.ContentMediaType;
+
+import java.util.Objects;
+
+public class PostRequestBody {
```

Comment:
Thinking thar `RawRequestBody` might be better since it could be used with `PUT`. Or  maybe I'm missing that `Post` means something else.

Reply:
Totally right there, for some reason was thinking about POST.  Will address that- I don;t like the word "Raw", because its not very raw in fact, but will propose another name.